crates/ruff_linter/src/rules/flake8_use_pathlib/rules/invalid_pathlib_with_suffix.rs

```diff
@@ -34,6 +36,9 @@ use ruff_text_size::Ranged;
 /// function parameter annotated with `Path` or from a direct assignment to a
 /// `Path()` constructor call.
 ///
+/// ## Limitations
+/// The single dot `"."` suffix is invalid and cannot be fixed by Ruff.
+///
```

Comment:
I think you can just move this inside the explanation of fix safety (maybe at the end), and say

```rust
/// No fix is offered if the suffix `"."` is given, since the intent is unclear.
```